test/e2e/fixture/app/actions.go

```diff
@@ -25,6 +25,11 @@ func (a *Actions) DeleteFile(file string) *Actions {
 	return a
 }
 
+func (a *Actions) AddFile(fileName, fileContents string) *Actions {
```

Comment:
The SyncFail Hook test required two hooks. Instead of adding one to the `testdata/hook` folder, I added this fixture to add files on the fly.

Reply:
Good stuff.